crates/transaction-pool/src/error.rs

```diff
@@ -323,7 +336,10 @@ impl InvalidPoolTransactionError {
                 }
             }
             Self::Eip7702(eip7702_err) => match eip7702_err {
-                Eip7702PoolTransactionError::MissingEip7702AuthorizationList => false,
+                Eip7702PoolTransactionError::MissingEip7702AuthorizationList => true,
```

Comment:
let's add a comment for why this should be `true`, given we would always reject these transactions in the pool, I think it's fine for it to be `true`